app/src/androidTest/java/com/github/wnder/NavigationActivityTest.java

```diff
@@ -104,14 +104,6 @@ public void guessButtonWithoutConnectionLaunchesAlert() {
         onView(withText(R.string.no_connection)).check(matches(isDisplayed()));
         when(networkInfo.isNetworkAvailable()).thenReturn(true);
     }
-
-    @Test
-    public void tourButtonGoesToTour() {
-        onView(withId(R.id.guess_page)).perform(click());
-        onView(withText("Radius: 5km")).check(matches(isDisplayed()));
-        onView(withId(R.id.navigationToTourButton)).perform(click());
-        Intents.intended(hasComponent(GuessPreviewActivity.class.getName()));
-    }
```

Comment:
Any reason for removing this test instead of changing the expected activity?

Reply:
nope just forgot